.github/workflows/check.yml

```diff
@@ -39,6 +39,4 @@ jobs:
           toolchain: ${{ matrix.toolchain }}
           components: clippy
       - name: cargo clippy
-        uses: actions-rs/clippy-check@v1
```

Comment:
This package has been deprecated, so I'm following the recommendation on https://doc.rust-lang.org/nightly/clippy/continuous_integration/github_actions.html